style.css

```diff
@@ -118,6 +118,56 @@ header {
   font-size: 20px;
 }
 
+/* Mobile-Menu */
+.hamburger {
+  position: fixed;
+  z-index: 100;
+  top: 1rem;
+  right: 1rem;
+  padding: 4px;
+  border: #fff solid 1px;
+  background: inherit;
+  cursor: pointer;
+}
+
+.closeIcon {
+  display: none;
+  color: #fff;
+  background-color: #6070ff;
+  border: #6070ff solid 10px;
+}
+
+.menu {
+  position: fixed;
+  transform: translateY(-100%);
+  transition: transform 0.2s;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: 99;
+  background: #6070ff;
+  opacity: 0.9;
+  color: white;
+  list-style: none;
+  padding-top: 4rem;
+}
```

Comment:
## [ OPTIONAL ]

- [ ] I would recommend sticking to the design as much as you can. Kindly add the bluer effect on the background of the mobile menu like the template below 👇 

The template | your design 
--------------|----------
![image](https://user-images.githubusercontent.com/65068771/147004330-77a63000-8077-414e-a544-2b46e4a7d1d2.png) | ![image](https://user-images.githubusercontent.com/65068771/147004511-45d38701-8745-4d5f-ad6f-96aeb3be7d6d.png)